# def add(n1,n2):
#     return n1 + n2 

# add(1,input("Num2:"))
# # output: error

# try:
#     add(1,input("Num2:"))
# except:
#     print("Error")
# else: 
#     print("excecute if except is not excecuted")
# finally:
#     print("Done")
